tests/block_tools.py

```diff
@@ -257,25 +283,35 @@ def change_config(self, new_config: Dict):
             save_config(self.root_path, "config.yaml", self._config)
 
     def add_plot_directory(self, path: Path) -> None:
-        self._config = add_plot_directory(self.root_path, str(path))
+        try:
+            self._config = add_plot_directory(self.root_path, str(path))
+        except ValueError:
+            if not self.automated_testing:  # ignore if this is user run
+                pass
+            else:
+                raise
 
-    async def setup_plots(self):
+    async def setup_plots(self, plots: Optional[int] = None, plot_size: int = 20):
+        if plots is not None:
```

Comment:
Use the default argument value instead:

`ansyc def setup_plots(self, num_og_plots: int=15, num_pool_plots: int=5 ...)`

Reply:
I did that to try to avoid using another variable, however I'll just use self.automated_testing

Reply:
Done, thanks!